pkg/gatewayserver/io/ws/lbslns/downstream.go

```diff
@@ -88,8 +85,8 @@ func (f *lbsLNS) FromDownlink(ctx context.Context, uid string, down ttnpb.Downli
 	// Estimate the xtime based on the timestamp; xtime = timestamp - (rxdelay). The calculated offset is in microseconds.
 	dnmsg.XTime = xTime - int64(dnmsg.RxDelay*int(time.Second/time.Microsecond))
 
-	// This field is not used but needs to be defined for the station to parse the json.
-	dnmsg.DevEUI = "00-00-00-00-00-00-00-00"
+	// This field is not used but needs to be defined for the station to parse the json and should be non-zero for the station to return tx confirmations.
+	dnmsg.DevEUI = "00-00-00-00-00-00-00-01"
```

Comment:
WTF, really?

Reply:
`DevEUI` and `Diid`, but indeed, silly.

Reply:
Yup. I've summarized the behaviour here; https://www.thethingsindustries.com/docs/reference/lbs-implementation-guide/implementation-details/#use-of-deveui-field-in-downlink-messages